### ISSUE TITLE
[alpha2] eventLimit & multidays event on month view

### ISSUE THREAD

**Issue by frlinw** [fullcalendar/fullcalendar#4331]
### Bug Reports

Repro: https://codepen.io/frlinw/pen/pxgmYr

1) Friday 19: does not respect eventLimit
2) Friday 19: Click on `+1 more` link generate the following error:
`Uncaught TypeError: Cannot read property 'eventRange' of null`

Look at the console for more information.

(I didn't try this case on v3)

**Comment by acerix** [comment]
Thanks for the report.  I get this error in Firefox.
```
TypeError: s is null
e/this.handleSegClick
https://rawgit.com/fullcalendar/fullcalendar/v4.0.0-alpha.2/dist/fullcalendar.min.js:11:8553
i
https://rawgit.com/fullcalendar/fullcalendar/v4.0.0-alpha.2/dist/fullcalendar.min.js:6:30572
```

**Comment by frlinw** [comment]
If an event is multiday and hidden on its start day, it seems it's always hidden... and it makes sense but look like a bug (and indeed there is an error). There is probably no perfect solution.

**Comment by arshaw** [comment]
i just pushed a fix to the v4 branch that gets rid of the JS error.

the behavior of how the events are laid out is actually intended, though up for debate. see/follow this ticket:
https://github.com/fullcalendar/fullcalendar/issues/4076

**Comment by arshaw** [comment]
this has been addressed and released in [4.0.0-alpha.4](https://github.com/fullcalendar/fullcalendar/releases/tag/v4.0.0-alpha.4)
